doc/guides/dts/index.rst

```diff
@@ -284,7 +354,7 @@ The following is a more precise list of required files:
 * Overlay Files (optional)
 
   * Overlay files contain tweaks or changes to the SoC and Board support files
-    described above. They can be used to modify Device Tree configurations
+    described above. They can be used to modify Devicetree configurations
```

Comment:
Maybe `Devicetree` could be consistently lowercase too, except for titles (e.g. `Devicetree specification`) and stuff. Less thinking required.